### DISCUSSION TITLE
Paralle Consumer on Openshift/Kubernetes Environment

### DISCUSSION THREAD

**Discussion by GokcerBelgusen** [confluentinc/parallel-consumer#810] [Q&A] [answered]
Is it possible to run Parallel Consumer on openshift or kubernetes environment on multi-pods ?

**Comment by rkolesnev** [comment]
Hi @GokcerBelgusen - can you elaborate ? 
I am tempted to say yes - similar like any other Kafka Consumer based application, the instances of parallel consumer can share consumer group and be scaled up and down in similar way to vanilla Kafka Consumer.

**Reply by GokcerBelgusen** [reply, marked as answer]
### Thank you,  I've tried, it works perfect.

**Comment by GokcerBelgusen** [comment]
Closing the topic
